src/file.helpers.ts

```diff
@@ -21,6 +21,7 @@ export function loadCliConfig(packageJson = loadPackageJson()): CliConfig {
   const leanixReportingCli = packageJson.leanixReportingCli || {};
 
   return {
+    srcPath: leanixReportingCli.srcPath ?? './src',
```

Comment:
I guess `'./src'` is the same as `getProjectDirectoryPath('src')`, which was the one previously used, right?

Reply:
Yes, `getProjectDirectoryPath()` is just the current working directory.